### PR TITLE
Remove less in favor of inline styles

### PR DESCRIPTION
**Pull request by ruyadorno**
Fear not my friends, this is a long but rather simple PR:

- Removed `less` package and any use of it in the app
- Styles are now part of the javascript
- Updated **material-ui** to v0.14.2
- Compatibility warning now uses new (and much better suited) api
- Removed `classnames` package

This brings us to a much better and stable version of `material-ui`, hopefully this will open the way to many ui fixes and improvements :tada: 

Feel free to review and merge @stefanbuck

### CODE REVIEW

**Comment by ruyadorno** [comment]
hi @mfunkie, you're also welcome to help review `renderer` changes :blush: 

a :+1: LGTM from you is good enough for me to merge :muscle:

**Comment by mfunkie** [comment]
I'll check it out in a couple of hours when I get a free second.

File: src/renderer/components/prompts/folder.jsx [hunk 1]
```diff
@@ -23,25 +22,21 @@ export default React.createClass({
...
-      margin: '56px 24px 0px 24px'
-    };
-
     const isEmpty = !!this.props.selectedFolder;
 
     return (
-      <div className="select-folder-prompt fieldset">
-        <label style={{ background: this.props.color }}>
+      <div style={this.getContainerStyle()}>
+        <label style={this.getLabelStyle()}>
```
Comment (mfunkie) [inline src/renderer/components/prompts/folder.jsx]: This is just a preference, but I would much rather see a `<Label />` component than a mixin that adds functions to the component to get styles when that function always returns the same styles.  (This goes for Container as well)

Comment (mfunkie) [inline src/renderer/components/prompts/folder.jsx]: The [React Native Web](https://github.com/necolas/react-native-web) project has some great [examples](https://github.com/necolas/react-native-web/blob/master/docs/components/Text.md#examples) of building abstractions into components for styling.

Comment (ruyadorno) [inline src/renderer/components/prompts/folder.jsx]: :+1: makes a lot of sense! thanks :smile:

Comment (stefanbuck) [inline src/renderer/components/prompts/folder.jsx]: Fantastic, @mfunkie thank you for your feedback. @ruyadorno are you planning to refactor it within this PR?

Comment (ruyadorno) [inline src/renderer/components/prompts/folder.jsx]: :+1: yep, hope to get to it soon™

**Comment by ruyadorno** [comment]
:+1: done!

many thanks @mfunkie, composing separated components is definitely much better than stuffing everything inside a mixin :tada: 

@stefanbuck we're good to :shipit:

**Comment by mfunkie** [comment]
:+1: Nice job!

**Comment by stefanbuck** [comment]
Great job :+1:

**Pull request merged**